src/BlazorWebFormsComponents/JavaScriptConstants.tt

```diff
@@ -0,0 +1,46 @@
+﻿<#@ template debug="false" hostspecific="true" language="C#" #>
+<#@ output extension=".cs" #>
+<#@ assembly name="System.IO"#>
+<#@ import namespace="System.IO" #>
+<#@ import namespace="System.Text" #>
+<#@ import namespace="System.Collections" #>
+<#@ import namespace="System.Collections.Generic" #>
+
+namespace BlazorWebFormsComponents
+{
+  /// <summary>
```

Comment:
Nit:  spaces vs. tab here has the first row mis-aligned with the rest of the comment

Reply:
Fixed, I think. Changed the spaces to tabs. 
Would be a good idea to add an .editorconfig file to help with those kinds of style differences.